Frontend/components/orders/ordersTable.js

```diff
@@ -25,43 +25,40 @@ const OrdersTable = () => {
         
         <>
             <Title>Recent Orders</Title>
-            {!isSSR && orderStatus && orderDetailStatus === 'loading' && (
+            {!isSSR && orderStatus === 'loading' && (
                 <Typography>Loading data...</Typography>
             )}
-            {!isSSR && orderStatus && orderDetailStatus === 'error' && (
+            {!isSSR && orderStatus === 'error' && (
                 <Typography>Error fetching data</Typography>
             )}
 
-            {!isSSR && orderStatus && orderDetailStatus === "success" && (
+            {!isSSR && orderStatus === "success" && (
             <>
                 <Table size="small">
                     <TableHead>
                         <TableRow>
                             <TableCell>ID</TableCell>
                             <TableCell>Customer ID</TableCell>
                             <TableCell>Customer Name</TableCell>
-                            {/* <TableCell>Product</TableCell> */}
                             <TableCell>Date</TableCell>
                             <TableCell>Ship To</TableCell>
-                            <TableCell align="right">Quantity</TableCell>
                             <TableCell align="right">Invoice Amount</TableCell>
+                            <TableCell></TableCell>
                         </TableRow>
                     </TableHead>
                     <TableBody>
                         {orderData.map(order => (
                             <TableRow key={order["id"]}>
+                                <TableCell>{order["id"]}</TableCell>
                                 <TableCell>{order["customerId"]}</TableCell>
                                 <TableCell>{order["customerName"]}</TableCell>
-                                <TableCell>{order["orderDate"]}</TableCell>
+                                <TableCell>{Date(order["orderDate"]).toString()}</TableCell>
                                 <TableCell>{order["shipTo"]}</TableCell>
                                 <TableCell align="right">{`$${order["invoiceAmount"]}`}</TableCell>
                             </TableRow>
                         ))}
                     </TableBody>
                 </Table>
-                <Link color="primary" href="#" onClick={preventDefault} sx={{ mt: 3 }}>
-                    See more orders
-                </Link>
```

Comment:
Good that you take it out! 💯

Reply:
<img width="683" alt="image" src="https://user-images.githubusercontent.com/89083753/174274300-c81b6b1f-0bad-4438-81ef-07192dc3e524.png">
Can I do that now?